src/utils/certificates.test.ts

```diff
@@ -13,7 +13,10 @@ let writeSecretWithObjectSpy: jest.SpyInstance
 const config = {
   common_name: 'me',
   country: 'us',
-  organization: 'rbhe'
+  organization: 'rbhe',
+  mps_tls_config: {
+    minVersion: 'TLS1'
```

Comment:
what happens if you enter a non supported option? Also, what is the list of supported options?

Reply:
Hi Mike,
TLSv1, TLSv1.1, TLSv1.2, TLSv1.3 are supported. if unsupported there will be error

Reply:
You want to put those in the documentation?